aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/asf/BuildDataCollector.kt

```diff
@@ -22,7 +22,7 @@ import android.os.Build
  * Collects build information for underlying device hardware.
  */
 class BuildDataCollector : DataCollector {
-    companion object {
+    internal companion object {
```

Comment:
Can we make the BuildDataCollector internal instead?

Reply:
Yes this is a mistake